templates/springboot/cloudevents/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>3.1.5</version>
+    <version>3.2.1</version>
```

Comment:
Just FYI, there is a brand new 3.2.2 release of Spring Boot now

Reply:
We might keep this to check if auto-updater changes this.